CMakeLists.txt.user

```diff
@@ -0,0 +1,345 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be in repo.

Add it to .gitignore, please (separate commit).

Reply:
Added.